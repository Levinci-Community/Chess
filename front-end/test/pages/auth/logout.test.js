import React from "react";
import { render } from "@testing-library/react";
import LogoutPage from "../../../src/pages/auth/logout";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({
    t: (key) => key,
  }),
}));

test("renders LogoutPage snapshot", () => {
    const { container } = render(
      <Router>
        <LogoutPage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
