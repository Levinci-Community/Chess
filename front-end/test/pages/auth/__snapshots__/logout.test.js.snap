// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LogoutPage snapshot 1`] = `
<div>
  <div
    class="css-fdfpf0"
  >
    <h2
      class="chakra-heading css-hvj0dp"
    >
      auth.logging_out
    </h2>
    <div
      class="chakra-spinner css-1y7joxr"
    >
      <span
        class="css-8b45rq"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
